lib/Sabre/PropFindPlugin.php

```diff
@@ -91,7 +93,7 @@ public function initialize(Server $server) {
 	}
 
 	public function propFind(PropFind $propFind, INode $node): void {
-		if ($node instanceof AlbumPhoto) {
+		if ($node instanceof AlbumPhoto || $node instanceof LocationPhoto) {
```

Comment:
```suggestion
		if ($node instanceof CollectionPhoto) {
```
?

Reply:
Nice indeed!

Reply:
Actually no, some methods are not declared on `CollectionPhoto`, but only by its children.